docker-compose.yml

```diff
@@ -36,6 +50,8 @@ services:
       depends_on:
         - db
         - redis
+        - extractor
+      hostname: server
```

Comment:
Defining hostname is redundant here as it will be the service name which is what the value is.